xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4391,6 +4418,9 @@ bool CVideoPlayer::OnAction(const CAction &action)
       }
       else
         break;
+    case ACTION_TOGGLE_COMMSKIP:
+      ToggleCommSkip();
```

Comment:
> Why don't you disable the function by default and have users enable it with the toggle on demand? THat would eliminate the need for a setting.

Disabling the function by default **changes the current behavior** of the player which has been using the EDL files to cut, mute, or skip over indicated sections of all files being played for quite some time now. As mentioned in another message, this functionality pre-dates PVR functionality in Kodi/XBMC. The original reason for adding a toggle to disable this function is that occasionally the EDL files which are generated are "corrupt." (Technically, they aren't corrupt since they are valid files, they just have very wrong time points for these cuts/mutes/skips). In those cases, it is desirable to **temporarily disable** the automatic actions for that single, specific player session.

I don't see why adding one more EDL configuration option to the existing set of them in the advancedsettings is a problem. But if that must be avoided, then the default should be for EDL processing to be enabled so that the current functionality of the player is not changed.